tests/test_oneline_parser.py

```diff
@@ -25,6 +25,54 @@
                 "end_column": 17,
             },
         ),
+        # Test case for leading space after start sequence (bug fix)
```

Comment:
The test cases seem not covering the case where Kilian mentioned. It would be nice to have a separate test case with `field_split_char` set to `a space` instead of the default comma `,` as shown in his example:

`@Implementation <field1> <field2>`